annotator-scanner/src/main/java/edu/ucr/cs/riple/scanner/location/MethodLocation.java

```diff
@@ -45,7 +46,7 @@ public String tabSeparatedToString() {
         "\t",
         type.toString(),
         enclosingClass.flatName(),
```

Comment:
Should we do this, for consistency?
```suggestion
        Serializer.serializeSymbol(enclosingClass),
```

Reply:
Sure, we can. I will update it for all locations.

Reply:
@msridhar Updated in all subclasses, looks much better now.